### ISSUE TITLE
[Star Wars: Battlefront 2] Mouse stuck in top left corner (6060)

### ISSUE THREAD

**Issue by ibrokemypie** [ValveSoftware/Proton#223]
When launching battlefront 2 (a whitelisted game the mouse is stuck in the top left corner of the screen. Tabbing in and out leads to a black screen. There is no sound (though this could be due to being stuck on the profile select screen)

Tested on a arch with mesa-git (18.3) and a vega 56

**Comment by d10sfan** [comment]
Also seeing this on Ubuntu 18.04. 

Strangely, this worked on another installation of Ubuntu yesterday on the same system, so maybe a regression since a recent update?

**Comment by ibrokemypie** [comment]
yes, this works fine for me in a separate wine prefix (wine staging 3.12)

**Comment by intruden313** [comment]
Seeing this as well on Arch

**Comment by kforney** [comment]
Seeing this on Fedora 28 as well.
